R/dropdown.R

```diff
@@ -72,6 +72,60 @@ dropdown <- function(name, choices, choices_value = choices,
     )
 }
 
+#' Create a select list input control
+#'
+#' Create a select list that can be used to choose a single or multiple items from a list of values.
+#'
+#' @param inputId The input slot that will be used to access the value.
+#' @param label Display label for the control, or NULL for no label.
+#' @param choices List of values to select from. If elements of the list are named,
+#'   then that name — rather than the value — is displayed to the user.
+#' @param selected The initially selected value (or multiple values if multiple = TRUE).
+#'   If not specified then defaults to the first value for single-select lists and no
+#'   values for multiple select lists.
+#' @param multiple Is selection of multiple items allowed?
+#' @param width The width of the input.
+#' @param ... Arguments passed to \link{dropdown}.
+#'
+#' @export
+selectInput <- function(inputId, label, choices, selected = NULL, multiple = FALSE, width = NULL, ...) {
```

Comment:
Here I noticed inconsistency, in a whole lot of function we use name for id. I'm also down for `inputId` it should be nromalised at some point